src/_damon.py

```diff
@@ -828,12 +839,15 @@ def from_kvpairs(cls, kv):
                     if 'quotas' in kv else DamosQuotas(),
                 DamosWatermarks.from_kvpairs(kv['watermarks'])
                     if 'watermarks' in kv else DamosWatermarks(),
+                kv['target_nid'] if 'target_nid' in kv else None,
                 filters,
                 None, None)
 
     def to_kvpairs(self, raw=False):
         kv = collections.OrderedDict()
         kv['action'] = self.action
+        if is_damos_migrate_action(self.action):
+            kv['target_nid'] = self.target_nid
```

Comment:
We need to decide how to handle for non migrate actions. It currently doesn't print `target_nid` field, but we might have to set it to `-1` explicitly to reset the value when unused.

Reply:
I think it's ok to do nothing, since the actions will simply ignore the value.

Reply:
Then it's fine as is. I wrote this part but thinking it again, I just felt uncomfortable having a stale value. Let's leave it.